extensions/script-libraries/drawing/drawing.livecodescript

```diff
@@ -0,0 +1,2960 @@
+﻿script "com.livecode.library.drawing"
+/*
+Copyright (C) 2017 LiveCode Ltd.
+
+This file is part of LiveCode.
+
+LiveCode is free software; you can redistribute it and/or modify it under
+the terms of the GNU General Public License v3 as published by the Free
+Software Foundation.
+
+LiveCode is distributed in the hope that it will be useful, but WITHOUT ANY
+WARRANTY; without even the implied warranty of MERCHANTABILITY or
+FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
+for more details.
+
+You should have received a copy of the GNU General Public License
+along with LiveCode.  If not see <http://www.gnu.org/licenses/>.  */
+
+on extensionInitialize
+   if the target is not me then
+      pass extensionInitialize
+   end if
+   
+   insert the script of me into back
+   
+   if the environment contains "development" then
+      set the _ideoverride of me to true
+   end if
+   
+   svgSpecLoad
+end extensionInitialize
+
+on extensionFinalize
+   if the target is not me then
+      pass extensionFinalize
+   end if
+   
+   remove the script of me from back
+end extensionFinalize
+
+/**
+Title: Drawing Library
+
+Version: 0.0.0
+
+Author: LiveCode
+
+Description:
+This script library implements an SVG compiler for turning SVG files into
+'drawing metafiles'. A simple and efficient binary format for representing
+complex vector graphics, supported directly by the LiveCode image object.
+*/
+
+/******************************************************************************/
+
+/**
+Compile an SVG XML file to a drawing metafile.
+
+Example:
+on mouseUp
+	set the text of image 1 to drawingSvgCompileFile("clock.svg")
+end mouseUp
+
+Description:
+Use drawingSvgCompileFile to build a binary string representing an SVG file
```

Comment:
putting angle brackets round `<drawingSvgCompileFile>` will ensure it is displayed as the 'current entry' (currently in bold).

Reply:
Sure